src/ui/Templates/Template.ts

```diff
@@ -25,14 +30,24 @@ export class Template {
   constructor(public dataToString?: (object?: any) => string, public condition?: Function) {
   }
 
-  instantiateToString(object?: any, checkCondition = true): string {
+  /**
+   * Instantiate the template to a string if the condition matches
```

Comment:
Same as the previous class : I don't think we want to document this for external users.